webui/react/src/components/kit/Icon.tsx

```diff
@@ -96,8 +95,96 @@ export const IconNameArray = [
+      fill="currentcolor"
+      fillRule="evenodd"
+    />
+  </svg>
+);
+
+const FilterIcon: React.FC = () => (
+  <svg fill="none" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path
+      clipRule="evenodd"
+      d="M5 7.5C4.72386 7.5 4.5 7.72386 4.5 8C4.5 8.27614 4.72386 8.5 5 8.5H19C19.2761 8.5 19.5 8.27614 19.5 8C19.5 7.72386 19.2761 7.5 19 7.5H5ZM7.5 12C7.5 11.7239 7.72386 11.5 8 11.5H16C16.2761 11.5 16.5 11.7239 16.5 12C16.5 12.2761 16.2761 12.5 16 12.5H8C7.72386 12.5 7.5 12.2761 7.5 12ZM10.5 16C10.5 15.7239 10.7239 15.5 11 15.5H13C13.2761 15.5 13.5 15.7239 13.5 16C13.5 16.2761 13.2761 16.5 13 16.5H11C10.7239 16.5 10.5 16.2761 10.5 16Z"
+      fill="currentcolor"
+      fillRule="evenodd"
+    />
+  </svg>
+);
+
+const OptionsIcon: React.FC = () => (
+  <svg fill="none" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path
+      clipRule="evenodd"
+      d="M13.5 8C13.5 7.17157 14.1716 6.5 15 6.5C15.8284 6.5 16.5 7.17157 16.5 8C16.5 8.82843 15.8284 9.5 15 9.5C14.1716 9.5 13.5 8.82843 13.5 8ZM12.55 7.5H4C3.72386 7.5 3.5 7.72386 3.5 8C3.5 8.27614 3.72386 8.5 4 8.5H12.55C12.7816 9.64112 13.7905 10.5 15 10.5C16.2095 10.5 17.2184 9.64112 17.45 8.5H20C20.2761 8.5 20.5 8.27614 20.5 8C20.5 7.72386 20.2761 7.5 20 7.5H17.45C17.2184 6.35888 16.2095 5.5 15 5.5C13.7905 5.5 12.7816 6.35888 12.55 7.5ZM3.5 16C3.5 15.7239 3.72386 15.5 4 15.5H6.55001C6.78164 14.3589 7.79052 13.5 9 13.5C10.2095 13.5 11.2184 14.3589 11.45 15.5H20C20.2761 15.5 20.5 15.7239 20.5 16C20.5 16.2761 20.2761 16.5 20 16.5H11.45C11.2184 17.6411 10.2095 18.5 9 18.5C7.79052 18.5 6.78164 17.6411 6.55001 16.5H4C3.72386 16.5 3.5 16.2761 3.5 16ZM9 14.5C9.82843 14.5 10.5 15.1716 10.5 16C10.5 16.8284 9.82843 17.5 9 17.5C8.17157 17.5 7.5 16.8284 7.5 16C7.5 15.1716 8.17157 14.5 9 14.5Z"
+      fill="currentcolor"
+      fillRule="evenodd"
+    />
+  </svg>
+);
+
+const PanelIcon: React.FC = () => (
+  <svg fill="none" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path
+      clipRule="evenodd"
+      d="M10.72 5H19V19H10.72V5ZM9.72 5H5V19H9.72V5ZM10.72 4H19C19.5523 4 20 4.44772 20 5V19C20 19.5523 19.5523 20 19 20H10.72H9.72H5C4.93096 20 4.86356 19.993 4.79847 19.9797C4.34278 19.8864 4 19.4832 4 19V5C4 4.44772 4.44772 4 5 4H9.72H10.72Z"
+      fill="currentcolor"
+      fillRule="evenodd"
+    />
+  </svg>
+);
+
+const RowIconLarge: React.FC = () => (
+  <svg fill="none" height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg">
+    <path
+      clipRule="evenodd"
+      d="M5 6H19V14H5V6ZM4 6C4 5.44772 4.44772 5 5 5H19C19.5523 5 20 5.44772 20 6V14C20 14.5523 19.5523 15 19 15H5C4.44772 15 4 14.5523 4 14V6ZM4.5 18C4.22386 18 4 18.2239 4 18.5C4 18.7761 4.22386 19 4.5 19H19.5C19.7761 19 20 18.7761 20 18.5C20 18.2239 19.7761 18 19.5 18H4.5Z"
+      fill="currentcolor"
+      fillRule="evenodd"
+    />
+  </svg>
+);
+
+const RowIconMedium: React.FC = () => (
```

Comment:
Why do you need a new path for the icon to change size? Shouldn't you just be adjusting the height and width?

Reply:
Oh, these are not different sizes, they're different icons, I see.